recipes/cylc-flow/meta.yaml

```
@@ -0,0 +1,70 @@
+{% set name = "cylc-flow" %}
+{% set version = "8.0a1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "36f4f4363cb5cc7a3a613d2db6e84cc41c2825c6cd011191cc4821c5ca9c81a4"
```

I missed this first time, the quotes need to be removed to allow the update bot to work:
```suggestion
  sha256: 36f4f4363cb5cc7a3a613d2db6e84cc41c2825c6cd011191cc4821c5ca9c81a4
```

---

Thanks. Having looked at other ``meta.yaml`` files from a sample of recipes under ``conda-forge/feedstocks``, I am assuming you mean all lines with quotes, not just this one, so have removed them all (I'm about to push the changes up in the next commit).